apps/aeapi/src/aeapi.erl

```diff
@@ -434,6 +435,21 @@ generation_at_height(Height) when is_integer(Height) ->
             end
     end.
 
+balance_change_events_in_mempool_tx(SignedTx) ->
+    Tx = aetx_sign:tx(SignedTx),
+    Nonce = aetx:nonce(Tx),
+    SenderAccount = aetx:origin(Tx),
+    Txs = case Nonce of
+        0 ->
+            [SignedTx];
+        _ ->
+            {ok, UnsortedTxs} = aec_tx_pool:peek(infinity, SenderAccount, Nonce - 1),
+            lists:sort(fun(Left, Right) -> aetx:nonce(aetx_sign:tx(Left)) < aetx:nonce(aetx_sign:tx(Right)) end, [SignedTx|UnsortedTxs])
```

Comment:
Nonces starts at 1 (0 is reserved for, or at least only used by transactions wrapped in a GAMetaTx) - I don't think this logic is correctly handling GAMetaTx?

Reply:
The code is to avoid looking in the pool for transactions for an account if the nonce is 0. It should also check for 1. Otherwise it gets a list of transactions for a sender account for a dry run. There are issues with this, for example if there is a contract that has transactions from different accounts. The problem is that it is difficult to know what will happen until the transactions are mined. In the Rosetta documentation it says, "On this endpoint, it is ok that returned transactions are only estimates of what may actually be included in a block.", so I thought this should be good enough. I will do an example with a generalised account to see what happens.

Reply:
Ah, I didn't have the context... So you are trying to estimate how much assets an account will have once the transaction with nonce `Nonce` is included on chain? Well, there is no way to have a generic algorithm for ordering GA-transactions (they all have nonce 0), so skipping them is probably the right thing.

Reply:
This is only ever going to be an estimate, but as long as the user is aware of this I think this implementation is fine!